src/ota/ota_handler.rs

```diff
@@ -278,6 +279,9 @@ impl From<&OtaError> for OtaStatusMessage {
                 ota_status_message.message = message.to_string()
             }
             OtaError::Canceled => ota_status_message.status_code = "Canceled".to_string(),
+            OtaError::InconsistentState => {
+                ota_status_message.status_code = "InconsistentState".to_string()
```

Comment:
Are these also defined on the backend? Here I would return an internal error if that's the case

Reply:
https://docs.edgehog.io/0.9/astarte_interfaces.html#event-statuscode

Here the codes are documented.

```suggestion
                ota_status_message.status_code = "Internal".to_string();
                ota_status_message.message = ota_error.to_string(); 
```

Reply:
I agree with you, better to use something that it is already defined also on backed side.